docs/release-notes.rst

```diff
@@ -4,6 +4,12 @@
 Release Notes
 =============
 
+v2023.09.0 (September 7, 2023)
+------------------------------
+
+Bug fix in ``add_height_from_pressure_axis()`` and added example documentation.
```

Comment:
```suggestion
Bug Fixes
^^^^^^^^^^^^
* Unit and inferred height fixes in ``add_height_from_pressure_axis()``  by `Katelyn FitzGerald`_ in (:pr:`152`)

Documentation
^^^^^^^^^^^^^
* Additions to the examples gallery for ``find_local_extrema()``, ``plot_contour_labels()``, and ``plot_extrema_labels()`` by `Julia Kent`_ in (:pr:`145`)
```

Reply:
That sounds fine with me, but we'd still want a top-level plain text summary, right?